src/components/auth/login/Login.scss

```diff
@@ -0,0 +1,117 @@
+$color_1: #505e6c;
+$color_2: rgb(244, 71, 107);
+$color_3: inherit;
+$color_4: #6f7a85;
+$background_color_1: #ffffff;
+
+.login-clean {
+  background: #f1f7fc;
+  padding: 80px 0;
+  form {
+    max-width: 320px;
+    width: 90%;
+    margin: auto;
+    background-color: $background_color_1;
+    padding: 30px;
+    border-radius: 4px;
+    color: $color_1;
+    box-shadow: 1px 1px 5px rgba(0, 0, 0, 0.1);
+    .form-control {
+      background: #f7f9fc;
+      border: none;
+      border-bottom: 1px solid #dfe7f1;
+      border-radius: 0;
+      box-shadow: none;
+      outline: none;
+      color: $color_3;
+      text-indent: 8px;
+      height: 42px;
+    }
+    .btn-primary {
+      background: rgb(2, 56, 115);
+      border: none;
+      border-radius: 4px;
+      padding: 11px;
+      box-shadow: none;
+      margin-top: 26px;
+      text-shadow: none;
+      outline: none !important;
+      &:hover {
+        background: #023873;
+      }
+      &:active {
+        background: #eb3b60;
+        transform: translateY(1px);
+      }
+    }
+    .forgot {
+      display: block;
+      text-align: center;
+      font-size: 12px;
+      color: $color_4;
+      opacity: 0.9;
+      text-decoration: none;
+      &:hover {
+        opacity: 1;
+        text-decoration: none;
+      }
+      &:active {
+        opacity: 1;
+        text-decoration: none;
+      }
+    }
+  }
+  .illustration {
+    text-align: center;
+    padding: 0 0 20px;
+    font-size: 100px;
+    color: $color_2;
+  }
+}
+.form-group button {
+  background-color: rgb(16, 75, 137);
+}
+.a {
+  color: rgb(232, 237, 242);
+}
+.i {
+  color: rgb(16, 75, 137);
+}
+button.kep-login-facebook {
+  padding: 8px;
+  // font-size: 1px;
+}
```

Comment:
If this font size is of no use kindly remove it.

Reply:
removed.